content/en/tracing/trace_pipeline/ingestion_controls.md

```diff
@@ -92,72 +72,133 @@ Filter the page by environment, configuration, and status to view services for w
 {{< /callout >}}
 
-Click **Manage Ingestion Rate** to get instructions on how to configure your service ingestion rate.
+To configure sampling rates for the service by resource name: 
+1. Click **Manage Ingestion rate**.
+   {{< img src="/tracing/trace_indexing_and_ingestion/sampling_configuration_modal.png" alt="Configuration Modal" style="width:100%;">}}
+2. Click **Add new rule** to set sampling rates for some resources. Sampling rules use glob pattern matching, so you can use wildcards (`*`) to match against multiple resources at the same time.
+
+#### if the Remote configuration option is available
 
-{{< img src="tracing/trace_indexing_and_ingestion/service_ingestion_rate_config.png" style="width:100%;" alt="Change the Service Ingestion Rate" >}}
+3. Click **Apply** to save the configuration. 
 
-To specify a specific percentage of a service's traffic to be sent, add an environment variable or a generated code snippet to your tracing library configuration for that service.
+The configuration should take effect in less than a minute. You **do not need** to redeploy the service for the change to take effect. You can observe the configuration changes from the [Live Search Explorer][9].
 
-1. Select the service you want to change the ingested span percent for.
-2. Choose the service language.
-3. Choose the desired ingestion percentage.
-4. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. **Note**: The service name value is case sensitive. It should match the case of your service name.
-5. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The ingestion reason for the service is shown as `ingestion_reason:rule`.
+From the **Service Ingestion Summary**, resources for which the sampling rate are remotely applied should show as `Remote Configured` in the **Configuration** column.
+
+#### if the Remote configuration option is disabled
+
+If the remote configuration option is disabled, make sure that the listed [requirements](#remote-configuration-requirements) are all met to be able to use remote configuration.
+
+3. Apply the appropriate configuration generated from these choices to the indicated service and redeploy the service. 
+**Note**: The service name value is case sensitive. It should match the case of your service name.
+
+4. Confirm on the Ingestion Control Page that your new percentage has been applied by looking at the Traffic Breakdown column, which surfaces the sampling rate applied. The resources for which the sampling rate was applied should show as `Local Configured`.
```

Comment:
I think it would be easier to read if we put each of these paths into a tab or collapse-content shortcode. Let me know if you'd like help applying that!